docs/ucl_myriad.md

```diff
@@ -40,3 +42,27 @@ Then make sure that your bin PATH is executable, by placing the following line i
 ```bash
 export PATH=$PATH:/home/<YOUR_ID>/bin
 ```
+
+### Common troubleshooting issues:
+
+1. If your job gets terminated by SIGHUP, you should then submit your nextflow script within a job:
+
+`qsub -l h_rt=48:00:00 -l mem_free=40G -pe smp 1 -b y nextflow run`
```

Comment:
````suggestion
```bash
qsub -l h_rt=48:00:00 -l mem_free=40G -pe smp 1 -b y nextflow run
```
````
gives you a nice copy-code button